snapcraft/internal/pluginhandler.py

```diff
@@ -324,23 +324,9 @@ def migratable_fileset_for(self, step):
         return _migratable_filesets(fileset, self.code.installdir)
 
     def _organize(self):
-        organize_fileset = getattr(self.code.options, 'organize', {}) or {}
+        fileset = getattr(self.code.options, 'organize', {}) or {}
```

Comment:
Isn't `fileset = getattr(self.code.options, 'organize', {})` enough? The `or {}` is redundant, no?